### ISSUE TITLE
Move `build` to PyPA org on PyPI?

### ISSUE THREAD

**Issue by webknjaz** [pypa/build#667]
With the orgs feature on PyPI, there's a PyPA org that exists already — https://pypi.org/org/pypa/. Why not make use of it?

_UPD_: Dustin explained the implications as an FAQ in the ML: https://mail.python.org/archives/list/pypa-committers@python.org/thread/E6MWIHEK3M232UILXGQFYPHGJHF7VYW7/.

**Comment by pradyunsg** [comment]
See https://github.com/pypa/pip/issues/12250#issuecomment-1698025213 -- let's hold off on this.

**Comment by webknjaz** [comment]
In the other issue Henry said that he'd like to move.

**Comment by henryiii** [comment]
Yes, I've just started using orgs for Scikit-HEP and I think it's nice; we have a GitHub org, having a PyPI org matching it makes sense.

**Comment by webknjaz** [comment]
@henryiii so add @di as an owner?

**Comment by henryiii** [comment]
I was hoping for at least one other maintainer to +1. Specifically @gaborbernat, @layday, and @FFY00 are on PyPI, so +1 from them would be good. IMO, this is the same as the GitHub `pypa` org, so anything in the GitHub org might as well be in the PyPA org.

**Comment by layday** [comment]
I don't mind - making the project's affiliation to the PyPA clearer on PyPI sounds like a plus. I'm a fan of package namespaces personally, and this is as close as we can get.

**Comment by di** [comment]
Done!
